pkgs/top-level/all-packages.nix

```diff
@@ -13600,6 +13600,12 @@ in
 
   shhopt = callPackage ../development/libraries/shhopt { };
 
+  signond = libsForQt5.callPackage ../development/libraries/signond { };
```

Comment:
If these derivations provide libraries, then they need to go into `mkLibsForQt5` instead of the top level.

Reply:
Fixed.